### PR TITLE
Created serveral Tests for GET and POST routes

### PR DESCRIPTION
**Pull request by insivika**
1. Created routes for POST request
2. Created routes for GET requests
3. Pulled buisness logic out of route handlers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "start-dev": "nodemon server/server.js",
     "start": "node server/server.js",
     "seeder": "node server/seeder.js",
-    "test": "jest"
+    "test": "jest --detectOpenHandles"
```
Comment (allenhsieh) [inline package.json]: I don't think detectOpenHandles is needed, that is just for debugging something funky, are you having issues without it?

File: server/database/db.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 const mongoose = require('mongoose')
 
 // Loading DB
-//'mongodb://localhost/fireBnb-booking'
 
 const mongodbUri = 'mongodb://insivika:insivika123@ds139675.mlab.com:39675/firebnb-booking'
```
Comment (allenhsieh) [inline server/database/db.js]: your username and password is live, on public github

File: package.json (hunk 1 above)
Comment (insivika) [reply package.json]: @allenhsieh  havent taken out, let me check

File: server/database/db.js (hunk 2 above)
Comment (insivika) [reply server/database/db.js]: working on fixing that now.

**Pull request merged**